src/sql/workbench/services/tableDesigner/browser/tableDesignerComponentInput.ts

```diff
@@ -55,11 +56,13 @@ export class TableDesignerComponentInput implements DesignerComponentInput {
 	constructor(private readonly _provider: TableDesignerProvider,
 		public tableInfo: azdata.designers.TableInfo,
 		private _telemetryInfo: ITelemetryEventProperties,
+		private _objectExplorerContext: azdata.ObjectExplorerContext,
```

Comment:
This is optional everywhere else - make sure you're handling it correctly here if that's intentional

Reply:
added extra check for this during refreshing.